### PR TITLE
refactor(getSumCol): improving performance of the maxSumColIndex func…

### PR DESCRIPTION
**Pull request by JeffersonH44**
…tion.

### CODE REVIEW

File: benchmark/maxSumCol.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    var maxSum = -Infinity;
+    for(var i = 0; i < X.columns; ++i) {
+        var currentSum = X.getColumnVector(i).sum();
+        if(currentSum > maxSum) {
+            maxSum = currentSum;
+            maxIndex = i;
+        }
+    }
+    return maxIndex;
+}
```
Comment (maasencioh) [inline benchmark/maxSumCol.js]: is really necessary have the benchmark in the library?

Comment (targos) [inline benchmark/maxSumCol.js]: it's not in the library, just in the benchmark folder

**Comment by targos** [comment]
@JeffersonH44 
Can you post the result of the benchmark here for posterity ?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ml-pls",
-  "version": "0.0.4",
+  "version": "0.0.6",
```
Comment (targos) [inline package.json]: you shouldn't increase the version number here. It is a separate process to release the package

**Comment by JeffersonH44** [comment]
v1 x 133,974 ops/sec ±0.22% (97 runs sampled)
v2 x 21,856,978 ops/sec ±0.20% (97 runs sampled)
v3 x 21,728,375 ops/sec ±0.18% (101 runs sampled)

File: package.json (hunk 2 above)
Comment (targos) [inline package.json]: and why jump directly from 0.0.4 to 0.0.6 ?

**Comment by targos** [comment]
the commit message is too long. you can put "improve performance of maxSumColIndex"

**Comment by JeffersonH44** [comment]
Change the version number of npm package

**Comment by targos** [comment]
LGTM

**Pull request merged**